dask/dataframe/core.py

```diff
@@ -5268,7 +5268,11 @@ def pop(self, item):
         return out
 
     @derived_from(pd.DataFrame)
-    def nlargest(self, n=5, columns=None, split_every=None):
+    def nlargest(self, n=5, columns=no_default, split_every=None):
+        if columns is no_default:
+            raise TypeError(
+                "DataFrame.nlargest() missing required positional argument: 'columns'"
+            )
```

Comment:
This seems totally fine as `columns=None` appears to already raise a (less informative) error.

Reply:
Yep this didn’t change anything.